framework/assets/yii.validation.js

```diff
@@ -436,5 +436,32 @@ yii.validation = (function ($) {
         }
     }
 
+    function validateMimeType(mimeTypes, fileType)
+    {
+        for (var i=0; i<mimeTypes.length; i++) {
```

Comment:
`for (var i=0, len = mimeTypes.length; i<len; i++) {` is a bit faster way to iterate since check is executed each time.